pkg/ibmcsidriver/ibm_csi_driver.go

```diff
@@ -31,14 +31,14 @@ import (
 
 // IBMCSIDriver ...
 type IBMCSIDriver struct {
-	name          string
-	vendorVersion string
-	logger        *zap.Logger
-	region        string
-
```

Comment:
I would suggest to have this parameter MaxVolumesPerNode a part of CSINodeServer struct itself, rather than adding it here. Since, it is used only CSINodeServer